src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -157,7 +159,24 @@ def export_config(cmd,
 
     if not skip_features:
         # Get all Feature flags with matching label
-        if destination in ('file', 'appconfig'):
+        if destination == 'file':
+            if format_ == 'properties':
+                skip_features = True
+            else:
+                if not naming_convention:
+                    logger.warning("--naming-convention was not provided. Reserved keywords for feature management section will default to PascalCase.")
+                    naming_convention = 'pascal'
```

Comment:
nit:maybe log debug? or no need to log here as we already have the explanation for default naming convention

Reply:
logging for debugging sounds good.

---
In reply to: [361021844](https://github.com/Azure/azure-cli/pull/11637#discussion_r361021844) [](ancestors = 361021844)